packages/client/src/comp/util/vbmWording.ts

```diff
@@ -0,0 +1,32 @@
+import { AddressContainer } from '../../lib/unstated'
+
+type VbmWording =
+  | 'ballot-by-mail'
+  | 'vote-by-mail ballot'
+  | 'absentee ballot'
+  | 'early voting ballot'
+  | 'vote by mail'
+
```

Comment:
We need a functions for
- `vbmNoun`: 'absentee voting' | 'voting by mail' | ''early voting', 
- `vbmBallot`: 'absentee ballot' | 'vote-by-mail ballot' | ''early voting ballot', 
- `vbmAdjective`: 'absentee ballot' | 'vote-by-mail' | ''early voting ballot', 
- `vbmVerb`: 'vote absentee' | 'vote by mail' | ''vote early', 

It's OK to define only one by state and the others based on just handling the 3 separate cases.